docs/getstdhandle.md

```diff
@@ -77,6 +84,16 @@ Some applications operate outside the boundaries of their declared subsystem; fo
 
 Some applications may also vary their behavior on the type of inherited handle. Disambiguating the type between console, pipe, file, and others can be performed with [**GetFileType**](/windows/win32/api/fileapi/nf-fileapi-getfiletype).
 
+### Handle disposal
+
+It is not required to [**CloseHandle**](/windows/win32/api/handleapi/nf-handleapi-closehandle) when use is complete. The returned value is simply a copy of the value stored in the process table. The process itself is generally considered the owner of these handles and their lifetime. It is placed in the table on creation depending on the inheritance and launch specifics of the [**CreateProcess**](/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessw) call and will be freed when the process is destroyed.
+
+The case to manipulate the lifetime of these handles manually with **CloseHandle** is if intentionally trying to replace or block other parts of the process from using these handles going forward. Remember that `HANDLE` is a numerical value so another thread storing it locally will not necessarily be updated when the value from the table is updated with **SetStdHandle**. Closing the value will close it process wide and the next usage of the stored copy will encounter an error on **ReadFile** or **WriteFile** accordingly.
+
+Guidance for replacing would be to get the existing `HANDLE` from the table with **GetStdHandle**, use **SetStdHandle** to place a new `HANDLE` in that is opened with **CreateFile** (or a similar function), then to close the retrieved handle.
```

Comment:
* In the first sentence I feel like some sort of pronoun or something is missing.
* In the second sentence, "Remember" feels a bit condescending
* If possible it'd be awesome if the English could be ~simplified~ computerized, because the MSDN machine translation might choke on this.

Reply:
I added a bit of clarity to the first bit. 	
I cut "remember" from the second bit.
I improved wording on the 3rd bit a little, I think.

Beyond that, I'm hoping @mattwojo can edit for clarity if necessary.